pkg/logging/aws.go

```diff
@@ -1,9 +1,53 @@
 package logging
 
+import aws_logging "github.com/aws/smithy-go/logging"
+
 type AWSAdapter struct {
 	Logger Logger
 }
 
-func (l *AWSAdapter) Log(vars ...interface{}) {
+func (l *AWSAdapter) logFunc(cls aws_logging.Classification) func(...interface{}) {
```

Comment:
Guess we need to wire this one to the adapter's cache creation and have the s3 client use it.